nym-validator-rewarder/src/rewarder/mod.rs

```diff
@@ -292,6 +320,10 @@ impl Rewarder {
                     }
                     break;
                 }
+                _ = &mut scraper_cancellation, if !scraper_cancellation.is_terminated() => {
```

Comment:
I've seen this sorta thing before, and I recall it was a workaround then?

Reply:
was it? I honestly don't remember